persistence-modules/spring-data-jpa-repo/README.md

```diff
@@ -10,6 +10,8 @@ This module contains articles about repositories in Spring Data JPA
 - [Spring Data Composable Repositories](https://www.baeldung.com/spring-data-composable-repositories)
```

Comment:
Is 'Introduction To Spring Data JPA' article from spring-data-jpa-repo-2 missed here?

Reply:
Yes, you are right. Updated the pr.